person.rb

```diff
@@ -0,0 +1,35 @@
+require './nameable'
+
+# Parent class for teacher and student
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permision: true)
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @rentals = []
```

Comment:
- [ ] Kindly make the `@rentals` have _**setter**_ and _**getter**_ methods: `attr_accessor :rentals`